src/cpp/shared/spell/spell_constants.h

```diff
@@ -0,0 +1,12 @@
+#ifndef SPELL_CONSTANTS_H
+#define SPELL_CONSTANTS_H
+
+#pragma pack(push, 1)
```

Comment:
It's perfectly fine that this is here -- it stops us from forgetting it in case we add anything to the file.

But, I can tell you this doesn't do anything. I'd like to check that you understand why.